tests/e2e/e2e_setup_test.go

```diff
@@ -55,7 +55,7 @@ const (
 
 var (
 	stakeAmount       = math.NewInt(100000000000)
-	stakeAmountCoin   = sdk.NewCoin("stake", stakeAmount)
+	stakeAmountCoin   = sdk.NewCoin(uatomDenom, stakeAmount)
```

Comment:
Mismatch in denoms? uatom/stake?

Reply:
I changed the bonding denom to uatom to keep everything in the test consistent...tweaked the variable names to reflect it's the amount for staking not a stake denom amount.